drivers/adc/ad463x/iio_ad463x.c

```diff
@@ -328,7 +323,6 @@ static int32_t _iio_ad463x_read_dev(struct iio_ad463x *desc, uint32_t *buff,
 			buff[j++] = data[i];
 		}
 
-	ad463x_enter_config_mode(desc->ad463x_desc);
```

Comment:
is this change related ?

Reply:
this is from the patch by @jonathanns. ill answer from what i understood:

there is no iio fuction that actually needs config mode set, so we dont
set cfg mode after reading the raw data bytes. 

however at start of each function to read data, exit_cofig mode is called
just to make sure that the chip is in a good state to return data, and that
cfg mode was not set by any noise on the MOSI line.